src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -2046,7 +2050,7 @@ export function validateTerminalName(name: string): { content: string, severity:
 	return null;
 }
 
-function convertOptionsOrProfileToOptions(optionsOrProfile?: ICreateTerminalOptions | ITerminalProfile): ICreateTerminalOptions | undefined {
+function convertOptionsOrProfileToOptions(optionsOrProfile: ICreateTerminalOptions | ITerminalProfile): ICreateTerminalOptions {
```

Comment:
Keeping the undefined support can make this more convenient to use as `|| {}` can be avoided on each call. I think this needs to change back anyway to not map undefined to {} in the other comment.